src/components/Stratosphere/Footer.tsx

```diff
@@ -16,34 +16,38 @@ const FooterLink: VoidFunctionComponent<{ href: string; label: React.ReactNode }
   </TextContent>
 );
 
-const Footer = () => (
-  <div className="chr-c-footer">
-    <Level>
-      <LevelItem>
-        <img className="chr-c-footer__logo pf-v5-u-mr-3xl" src="/apps/frontend-assets/red-hat-logos/logo.svg" />
-      </LevelItem>
-      <LevelItem className="pf-v5-u-mr-2xl">
-        <TextContent>
-          <Text component="small">Copyright c 2023 Red Hat, Inc.</Text>
-        </TextContent>
-      </LevelItem>
-      <LevelItem className="pf-v5-u-mr-auto">
-        <Flex>
-          <FlexItem>
-            <FooterLink href="https://www.redhat.com/en/about/privacy-policy" label="Privacy statement" />
-          </FlexItem>
-          <Divider className="chr-c-footer__divider" inset={{ default: 'insetSm' }} orientation={{ default: 'vertical' }} />
-          <FlexItem>
-            <FooterLink href="https://www.redhat.com/en/about/terms-use" label="Terms of service" />
-          </FlexItem>
-          <Divider className="chr-c-footer__divider" inset={{ default: 'insetSm' }} orientation={{ default: 'vertical' }} />
-          <FlexItem>
-            <FooterLink href="https://www.redhat.com/en/about/all-policies-guidelines" label="All policies and guidelines" />
-          </FlexItem>
-        </Flex>
-      </LevelItem>
-    </Level>
-  </div>
-);
+const Footer = () => {
+  const currentYear = useRef(new Date().getFullYear());
```

Comment:
I don't think we need a ref here. You can just create the variable at the top of the file.